frontend - `npm install` yields peer dependency error

When I do `cd frontend; npm install`, I get:

``` bash
fontawesome#4.1.0 bower_components/fontawesome
npm ERR! peerinvalid The package karma does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer gulp-karma@0.0.4 wants karma@>=0.10 <=0.13
npm ERR! peerinvalid Peer karma-chai@0.1.0 wants karma@>=0.10.9
npm ERR! peerinvalid Peer karma-mocha@0.1.4 wants karma@>=0.12.8
npm ERR! peerinvalid Peer karma-phantomjs-launcher@0.1.4 wants karma@>=0.9
npm ERR! peerinvalid Peer karma-script-launcher@0.1.0 wants karma@>=0.9
npm ERR! peerinvalid Peer karma-chrome-launcher@0.1.4 wants karma@>=0.9.3
npm ERR! peerinvalid Peer karma-firefox-launcher@0.1.3 wants karma@>=0.9
npm ERR! peerinvalid Peer karma-html2js-preprocessor@0.1.0 wants karma@>=0.9
npm ERR! peerinvalid Peer karma-jasmine@0.1.5 wants karma@>=0.9
npm ERR! peerinvalid Peer karma-requirejs@0.2.2 wants karma@>=0.9
npm ERR! peerinvalid Peer karma-coffee-preprocessor@0.1.3 wants karma@>=0.9

npm ERR! System Darwin 13.2.0
npm ERR! command "/opt/local/bin/node" "/opt/local/bin/npm" "install"
npm ERR! cwd /Users/juhovepsalainen/Projects/github/js/geekprofile/frontend
npm ERR! node -v v0.10.29
npm ERR! npm -v 1.4.15
npm ERR! code EPEERINVALID
npm ERR!
npm ERR! Additional logging details can be found in:
npm ERR!     .../npm-debug.log
npm ERR! not ok code 0
```

--- a/frontend/package.json
+++ b/frontend/package.json
@@ -51,6 +51,6 @@
     "karma-script-launcher": "~0.1.0",
     "karma-chrome-launcher": "~0.1.2",
     "karma-firefox-launcher": "~0.1.3",
-    "karma": "~0.10.9"
+    "karma": "~0.12.6"
   }
 }

